Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -0,0 +1,1765 @@
+from typing import Dict, Any
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import sys
+import traceback
+import json
+import os
+import hashlib
+from datetime import timedelta
+from io import StringIO
+import logging
+import warnings
+import email
+from requests.exceptions import ConnectionError
+from collections import deque
+
+from multiprocessing import Process
+import exchangelib
+from exchangelib.errors import (
+    ErrorItemNotFound,
+    ResponseMessageError,
+    RateLimitError,
+    ErrorInvalidIdMalformed,
+    ErrorFolderNotFound,
+    ErrorMailboxStoreUnavailable,
+    ErrorMailboxMoveInProgress,
+    ErrorNameResolutionNoResults,
+    ErrorInvalidPropertyRequest,
+    MalformedResponseError,
+)
+from exchangelib.items import Item, Message, Contact
+from exchangelib.services.common import EWSService, EWSAccountService
+from exchangelib.util import create_element, add_xml_child, MNS, TNS
+from exchangelib import (
+    IMPERSONATION,
+    Account,
+    EWSDateTime,
+    EWSTimeZone,
+    Configuration,
+    FileAttachment,
+    Version,
+    Folder,
+    HTMLBody,
+    Body,
+    ItemAttachment,
+    OAUTH2,
+    OAuth2AuthorizationCodeCredentials,
+    Identity,
+)
+from oauthlib.oauth2 import OAuth2Token
+from exchangelib.version import EXCHANGE_O365
+from exchangelib.protocol import BaseProtocol, NoVerifyHTTPAdapter
+
+# Ignore warnings print to stdout
+warnings.filterwarnings("ignore")
+
+""" Constants """
+
+APP_NAME = "ms-ews-o365"
+
+# move results
+MOVED_TO_MAILBOX = "movedToMailbox"
+MOVED_TO_FOLDER = "movedToFolder"
+
+# item types
+FILE_ATTACHMENT_TYPE = "FileAttachment"
+ITEM_ATTACHMENT_TYPE = "ItemAttachment"
+ATTACHMENT_TYPE = "attachmentType"
+
+TOIS_PATH = "/root/Top of Information Store/"
+
+# context keys
+ATTACHMENT_ID = "attachmentId"
+ATTACHMENT_ORIGINAL_ITEM_ID = "originalItemId"
+NEW_ITEM_ID = "newItemId"
+MESSAGE_ID = "messageId"
+ITEM_ID = "itemId"
+ACTION = "action"
+MAILBOX = "mailbox"
+MAILBOX_ID = "mailboxId"
+FOLDER_ID = "id"
+
+# error messages
+SEARCH_MAILBOXES_RBAC = (
```

Comment:
this is not used

Reply:
You're right, this is related to a command that was removed 👍